Source/WEB_ROOT/admin/asset/assets.html

```diff
@@ -228,17 +218,16 @@ <h1>Asset Management</h1>
 	;]
 	    <tr>
 	        <td><a href="editAsset.html?assetid=~(id)" class="button"><em class="ui-icon-white ui-icon-pencil ui-icon"></em></a></td>
-	        <td><a href="assettag.html?assetid=~(id)" class="button">Tag</a></td>
 	        ~(view1;t;if.blank.then=

```

Comment:
Not part of your code review, but could this be converted to javascript? Simply thinking that if I were the next developer who came along, it would be nice to have the "actions" and "decision making" live within javascript. Unsure if that's possible...